### PR TITLE
[tests-only] [full-ci] updating given step for sharing in unlockFiles

### PR DESCRIPTION
**Pull request by S-Panta**
## Related Issue

- Part of https://github.com/owncloud/ocis/issues/8717#event-12535134605

### CODE REVIEW

File: tests/acceptance/features/bootstrap/SharingNgContext.php [hunk 1]
```diff
@@ -269,6 +269,7 @@ public function sendShareInvitation(string $user, TableNode $table, string $file
 	 */
 	public function userHasSentTheFollowingShareInvitation(string $user, TableNode $table): void {
 		$response = $this->sendShareInvitation($user, $table);
+		//        print_r($response->getBody ()->getContents ());
```
Comment (grgprarup) [inline tests/acceptance/features/bootstrap/SharingNgContext.php:272]: do we need this??

File: tests/acceptance/features/bootstrap/WebDavLockingContext.php [hunk 2]
```diff
@@ -351,8 +351,13 @@ public function publicHasLockedFileLastSharedFolder(
 		string    $file,
 		TableNode $properties
 	) {
+		if ($this->featureContext->isUsingSharingNG()) {
+			$token = $this->featureContext->shareNgGetLastCreatedLinkShareToken();
+		} else {
+			$token = $this->featureContext->getLastCreatedPublicShareToken();
+		}
```
Comment (grgprarup) [inline tests/acceptance/features/bootstrap/WebDavLockingContext.php:358]:
```suggestion
		$token = ($this->featureContext->isUsingSharingNG()) ? $this->featureContext->shareNgGetLastCreatedLinkShareToken() : $this->featureContext->getLastCreatedPublicShareToken();
```

File: tests/acceptance/features/bootstrap/WebDavLockingContext.php [hunk 3]
```diff
@@ -671,7 +676,11 @@ public function unlockItemAsPublicWithLastLockOfUserAndItemUsingWebDavAPI(
 		string $lockOwner,
 		string $itemToUseLockOf
 	) {
-		$user = $this->featureContext->getLastCreatedPublicShareToken();
+		if ($this->featureContext->isUsingSharingNG()) {
+			$user = $this->featureContext->shareNgGetLastCreatedLinkShareToken();
+		} else {
+			$user = $this->featureContext->getLastCreatedPublicShareToken();
+		}
```
Comment (grgprarup) [inline tests/acceptance/features/bootstrap/WebDavLockingContext.php:683]:
```suggestion
		$token = ($this->featureContext->isUsingSharingNG()) ? $this->featureContext->shareNgGetLastCreatedLinkShareToken() : $this->featureContext->getLastCreatedPublicShareToken();
```
`$user` or `$token`?

File: tests/acceptance/features/bootstrap/WebDavLockingContext.php [hunk 4]
```diff
@@ -690,7 +699,11 @@ public function unlockItemAsPublicWithLastLockOfUserAndItemUsingWebDavAPI(
 	 * @return void
 	 */
 	public function unlockItemAsPublicUsingWebDavAPI(string $itemToUnlock) {
-		$user = $this->featureContext->getLastCreatedPublicShareToken();
+		if ($this->featureContext->isUsingSharingNG()) {
+			$user = $this->featureContext->shareNgGetLastCreatedLinkShareToken();
+		} else {
+			$user = $this->featureContext->getLastCreatedPublicShareToken();
+		}
```
Comment (grgprarup) [inline tests/acceptance/features/bootstrap/WebDavLockingContext.php:706]:
```suggestion
		$user = ($this->featureContext->isUsingSharingNG()) ? $this->featureContext->shareNgGetLastCreatedLinkShareToken() : $this->featureContext->getLastCreatedPublicShareToken();
```
`$user` or `$token`?

File: tests/acceptance/features/bootstrap/WebDavLockingContext.php (hunk 2 above)
Comment (S-Panta) [reply tests/acceptance/features/bootstrap/WebDavLockingContext.php:358]: I guess previous looked cleaner than this one.

File: tests/acceptance/features/bootstrap/WebDavLockingContext.php (hunk 3 above)
Comment (S-Panta) [reply tests/acceptance/features/bootstrap/WebDavLockingContext.php:683]: somewhere token is used and somewhere user was used in step definition. So I added whatever was in previous code

**Review by saw-jan — APPROVED** [review]

**Pull request merged**